common/tasks/interleaved_weighted_round_robin.go

```diff
@@ -306,17 +325,30 @@ func (s *InterleavedWeightedRoundRobinScheduler[T, K]) doDispatchTasksWithWeight
 	channels WeightedChannels[T],
 ) {
 	numTasks := int64(0)
+	needChannelCleanup := false
 LoopDispatch:
 	for _, channel := range channels {
 		select {
 		case task := <-channel.Chan():
+			channel.UpdateLastActiveTime(s.ts.Now())
```

Comment:
nit: we can use the same now() value

Reply:
yeah I mean moving it above the for loop since we don't need a precise time here.

Reply:
Got it! Done.